src/OmnichannelChatSDK.ts

```diff
@@ -158,11 +162,65 @@ class OmnichannelChatSDK {
         return this.liveChatConfig;
     }
 
+    public async getChatReconnectContext(optionalParams: IChatReconnectOptionalParams = {}):  Promise<IChatReconnectContext> {
+        const context: IChatReconnectContext = {
+            reconnectId: null,
+            redirectURL: null
+        }
+
+        if (this.isAuthenticated) {
+            try {
+                const reconnectableChatsParams: IReconnectableChatsParams = {
+                    authenticatedUserToken: this.authenticatedUserToken as string
+                }
+    
+                const reconnectableChatsResponse = await this.OCClient.getReconnectableChats(reconnectableChatsParams);
+    
+                if (reconnectableChatsResponse && reconnectableChatsResponse.reconnectid) {  
+                    context.reconnectId = reconnectableChatsResponse.reconnectid as string
+                }
+            } catch {
+                const exceptionDetails = {
+                    response: "GetPreviousActiveSessionFailed"
+                }
+    
+                throw Error(exceptionDetails.response);
+            }
+        } else {
+            try {
+                if (optionalParams.reconnectId) {
+                    const  reconnectAvailabilityResponse = await this.OCClient.getReconnectAvailability(optionalParams.reconnectId);
+        
+                    if (reconnectAvailabilityResponse && !reconnectAvailabilityResponse.isReconnectAvailable) {
+                        if (reconnectAvailabilityResponse.reconnectRedirectionURL) {
+                            context.redirectURL = reconnectAvailabilityResponse.reconnectRedirectionURL as string;
+                        } 
+                    } else {
+                            //Reconnect URL is not expired, set the reconnect id
+                            this.reconnectId = optionalParams.reconnectId as string;
```

Comment:
`getChatReconnectContext` should be a feedback method and we should not persist any chat reconnect data yet. We can just return the reconnectId when it's valid